api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_eac_data/create_offers.py

```diff
@@ -584,6 +585,89 @@ def create_offers_base_list(
+            totalStudents=12,
+            totalTeachers=2,
+            phoneNumber="0199000000",
+            requestedDate=datetime.utcnow(),
+        )
+        templates.append(template)
+        number = next(number_iterator)
+        template = educational_factories.CollectiveOfferTemplateFactory(
+            name=f"offer {number} pour {offerer.name} template request",
+            educational_domains=[next(domains_iterator)],
+            venue=next(venue_iterator),
+        )
+        educational_factories.CollectiveOfferRequestFactory(
+            collectiveOfferTemplate=template,
+            educationalInstitution=next(institution_iterator),
+            educationalRedactor__lastName=f"last name {number} {offerer.name}",
+            educationalRedactor__firstName=f"first name {number} {offerer.name}",
+        )
+        templates.append(template)
+        number = next(number_iterator)
+        template = educational_factories.CollectiveOfferTemplateFactory(
+            name=f"offer {number} pour {offerer.name} template request",
+            educational_domains=[next(domains_iterator)],
+            venue=next(venue_iterator),
+        )
+        educational_factories.CollectiveOfferRequestFactory(
+            collectiveOfferTemplate=template,
+            educationalInstitution=next(institution_iterator),
+            educationalRedactor__lastName=f"last name {number} {offerer.name}",
+            educationalRedactor__firstName=f"first name {number} {offerer.name}",
+        )
+        templates.append(template)
+        number = next(number_iterator)
+        template = educational_factories.CollectiveOfferTemplateFactory(
+            name=f"offer {number} pour {offerer.name} template request",
+            educational_domains=[next(domains_iterator)],
+            venue=next(venue_iterator),
+        )
+        educational_factories.CollectiveOfferRequestFactory(
+            collectiveOfferTemplate=template,
+            educationalInstitution=next(institution_iterator),
+            educationalRedactor__lastName=f"last name {number} {offerer.name}",
+            educationalRedactor__firstName=f"first name {number} {offerer.name}",
+        )
+        templates.append(template)
+        number = next(number_iterator)
+        template = educational_factories.CollectiveOfferTemplateFactory(
+            name=f"offer {number} pour {offerer.name} template request",
+            educational_domains=[next(domains_iterator)],
+            venue=next(venue_iterator),
+        )
+        educational_factories.CollectiveOfferRequestFactory(
+            collectiveOfferTemplate=template,
+            educationalInstitution=next(institution_iterator),
+            educationalRedactor__lastName=f"last name {number} {offerer.name}",
+            educationalRedactor__firstName=f"first name {number} {offerer.name}",
+        )
+        templates.append(template)
```

Comment:
Est-ce qu'on ne pourrait pas tout mettre dans un boucle for plutôt ? Avec un peu de Faker pour varier certains champs.

Reply:
D'apres Raphael, le but du jeux est que la sandbox soit modifiable le plus rapidement et simplement possible, du coup je lui avais déjà posé la question et sa réponse a été que pour cela nous ne devons pas faire de boucle